lib/model/util.go

```diff
@@ -59,3 +68,95 @@ func (d *deadlockDetector) Watch(name string, mut sync.Locker) {
 		}
 	}()
 }
+
+// moveforconflict renames a file to deal with sync conflicts
+func moveForConflict(name string, maxConflicts int) error {
```

Comment:
These all should now live with the folder as we no longer call them from the model